### ISSUE TITLE
Blogpost: Hoodies in NYC for 🗽 EmpireJS! 

### ISSUE THREAD

**Issue by jennwrites** [hoodiehq/camp#27]
This is a [starter issue](http://hood.ie/blog/starter-issues.html). It’s reserved for participants of [EmpireJS OSS Night](http://2016.empirejs.org/#oss-night) or [NYC, Meet the Hoodies](https://ti.to/hoodie/nyc-meet-the-hoodies-3)! 
### :octocat:  Your First Pull Request?

We would like to take the opportunity to invite someone who did not yet contribute to Hoodie to work on this issue. Follow [@YourFirstPR](https://twitter.com/yourfirstPR) on Twitter for more of these issues across many open source projects :)
### 🤔 What you will need to know

English, knowledge of markdown isn't necessary but it helps. ☺️ 
### 🐞 The Issue

The Hoodies (Gregor, Nick, Jenn, Nathan, Nolan) are in NYC this week for [EmpireJS](http://2016.empirejs.org/) and we need to write a blog post about the different events! 

We need a contributor to write a brief synopsis about the event and/or gather some photos of the event. :relaxed: 

Also if there are links from mentions on Twitter (from talks or hack events) please include those! 

To take on this blog post:
- [ ] Claim this issue (`comment below to claim or assign yourself`)
- [ ] Compose a blog post about the event
- [ ] Find the instructions for posting a blog post to the [Hoodie blog](http://hood.ie/blog/) in the [`hoodiehq/hood.ie` repo here](https://github.com/hoodiehq/hood.ie#blog). 
- [ ] Submit a pull request (PR) as part of the end of the blog process. 
- [ ] You are done :clap: comment below that your PR is ready for review :tada:

Ping us in the [Hoodie Chat](http://hood.ie/chat/) or on [Twitter](https://twitter.com/hoodiehq/) if you have any questions :)

**Comment by gr2m** [comment]
Here are some pictures I took from the event:
https://www.dropbox.com/sh/6clhn2cdhxitk4f/AABrhCqaYgliQCyXgqOksBBwa?dl=0

in a nutshell:
1. I Took the train from Boston to New York on Wednesday :)
2. [OSS Night](http://2016.empirejs.org/#oss-night) on Wednesday evening, lots of new contributors on Hoodie!
3. EmpireJS on Thursday / Friday. I gave my talk on "Welcoming Communities" on Thursday morning. In the evening we went to Staten Island to bar, the trip there is really nice, we passed the Statue of Liberty. Friday night was karaoke!
4. Saturday morning was [the 3rd edition of Meet the Hoodies in New York](https://ti.to/hoodie/nyc-meet-the-hoodies-3). Hosted by Nathan / meetup.com. Meetup also sponsored Pizza and drinks <3 The rooftop picture is also at Meetup, super nice! There were also group pictures taken, I think @renrutnnej has it
5. Saturday evening we had our Hoodie Camp Release party! I can find a picture for that

Would be cool if someone would like to write a summary of all that :)
